django/db/migrations/operations/base.py

```diff
@@ -85,6 +98,23 @@ def describe(self):
         """
         return "%s: %s" % (self.__class__.__name__, self._constructor_args)
 
+    def formatted_description(self):
+        """
+        Output the describe output in a more informative way.
+        """
+        description = self.describe()
+        if self.category == OperationCategory.ADDITION:
+            return "+ {}".format(description)
+        if self.category == OperationCategory.REMOVAL:
+            return "- {}".format(description)
+        if self.category == OperationCategory.ALTER:
+            return "~ {}".format(description)
+        if self.category == OperationCategory.PYTHON:
+            return "p {}".format(description)
+        if self.category == OperationCategory.SeparateDatabaseAndState:
+            return "? {}".format(description)
+        return "s {}".format(description)
```

Comment:
Only `+` is tested. Can we add a direct unit test? (similar to the way we test `describe()`)